docs/content/how-to-guides/configure-sources/configure-mssql-source/_index.md

```diff
@@ -3,49 +3,60 @@ type: "docs"
 title: "Connect to Microsoft SQL Server"
 linkTitle: "Connect to Microsoft SQL Server"
 weight: 40
-toc_hide: true
-hide_summary: true
-no_list: true
 description: >
     Learn how to configure a SQL Server Source to connect to Microsoft SQL Server
 ---
 
 The SQLServer Source enables Drasi connectivity to Microsoft SQL Server databases.
 
-#### Source Requirements
+## Data Model
+The SQL Source translates the relational data from change events to more closely resemble property graph data change events so that they can be processed by subscribed Continuous Queries. To achieve this, it represents table rows as graph Nodes, as follows:
+- Each row gets represented as a Node with the table columns as properties of the Node.
+- The Node is assigned an id the is a composite of the table id and the row's primary key. This is Node metadata, not a property of the Node.
+- The name of the table is assigned as a **Label** of the Node.
+
+The SQL Server Source **does not** interpret foreign keys or joins from the relational source, instead relying on the Source Join feature provided by Continuous Queries to mimic graph-style Relations between Nodes based on the values of specified properties. See the [Source Joins](/concepts/continuous-queries/#sources) topic in the [Continuous Queries](/concepts/continuous-queries) section for details. 
+
+
+## Requirements
+On the computer from where you will create the Source, you need the following software:
+- Change data capture must be enabled on the database and each table you wish to observe.  See the documentation on [configuring SQL Server for CDC](./setup-sql-server).
+- [Drasi CLI](/reference/command-line-interface/) 
+- [Azure CLI](https://learn.microsoft.com/en-us/cli/azure/) (If using Azure Managed Identities)
```

Comment:
The use of Azure managed identities is mentioned here, but I don't see any documentation how to authenticate using this approach below. Should we add an example for this

Reply:
It's not working currently, so I removed it from the docs until we can fix it.